### PR TITLE
Chore: upload assets when new tag is pushed

### PR DESCRIPTION
**Pull request by mscb402**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          secret_id: ${{ secrets.TENCET_COS_SECRET_ID }}
+          secret_key: ${{ secrets.TENCET_COS_SECRET_KEY }}
+          bucket: ${{ secrets.TENCET_COS_BUCKET }}
+          region: ap-nanjing
+     - name: Make latest Tag
+       uses: rickstaa/action-create-tag@v1
+       with:
+          force_push_tag: true
+          tag: "latest"
+          message: "Latest release version ${{ github.ref_name }}"
```
Comment (tokers) [inline .github/workflows/release.yaml:30]: Add the EOL symbol.

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+          args: |
+            upload -r ./assets/ /${{ github.ref_name }}/assets/ && upload -r ./scripts/ /${{ github.ref_name }}/scripts/ \
+            && upload -r ./assets/ /latest/assets/ && upload -r ./scripts/ /latest/scripts/
+          secret_id: ${{ secrets.TENCET_COS_SECRET_ID }}
+          secret_key: ${{ secrets.TENCET_COS_SECRET_KEY }}
+          bucket: ${{ secrets.TENCET_COS_BUCKET }}
+          region: ap-nanjing
+     - name: Make latest Tag
+       uses: rickstaa/action-create-tag@v1
+       with:
```
Comment (tokers) [inline .github/workflows/release.yaml:27]: What's this?

Comment (mscb402) [reply .github/workflows/release.yaml]: removed

File: .github/workflows/release.yaml (hunk 1 above)
Comment (mscb402) [reply .github/workflows/release.yaml]: done

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  push_assets:
+   runs-on: ubuntu-latest
+   steps:
+     - name: Checkout
+       uses: actions/checkout@v2
+     - name: Upload to COS
+       uses: zkqiang/tencent-cos-action@v0.1.0
+       with:
+          args: |
+            upload -r ./assets/ /${{ github.ref_name }}/assets/ && upload -r ./scripts/ /${{ github.ref_name }}/scripts/ \
```
Comment (nic-6443) [inline .github/workflows/release.yaml:19]: We can delete `./scripts` directory now, it already replaced with cloud-cli. cc @tokers

Comment (mscb402) [reply .github/workflows/release.yaml]: removed

**Review by tokers — APPROVED** [review]

**Review by Hazel6869 — APPROVED** [review]

**Review by nic-6443 — APPROVED** [review]

**Pull request merged**